front/components/home/menu/config.ts

```diff
@@ -34,6 +34,11 @@ const CareersMenuConfig: MenuConfig = {
 const AboutMenuConfig: MenuConfig = {
   title: "About",
   items: [
+    {
```

Comment:
IIRC, This will also add it in the main navigation menu at the top.

Reply:
![Screenshot 2024-05-22 at 12 11 39](https://github.com/dust-tt/dust/assets/1189312/e445a57f-f662-4566-92e3-79a3b43db233)
 LGTM as well - it moved the x link out of the header, but as it's still available in the footer I'd say it's fine

Reply:
Would be better to not have it on top no?

Reply:
Le me know where and I'll move 
Right below "blog"? If I move belove the X link it won't appear in the header. I don't have a specific pref.